web_src/js/markup/codecopy.js

```diff
@@ -0,0 +1,20 @@
+import {svgNode} from '../svg.js';
+const {copied, copy_link_error} = window.i18n;
+
+export function renderCodeCopy() {
+  const els = document.querySelectorAll('.markup .code-block code');
+  if (!els?.length) return;
+
+  const button = document.createElement('button');
+  button.classList.add('code-copy', 'ui', 'button');
+  button.setAttribute('data-success', copied);
+  button.setAttribute('data-error', copy_link_error);
```

Comment:
As above, the success/error message can use common default messages.

Reply:
Done